tools/distpartitioning/data_proc_pipeline.py

```diff
@@ -44,38 +34,15 @@ def log_params(params):
                      help='The schema of the graph')
     parser.add_argument('--num-parts', required=True, type=int,
                      help='The number of partitions')
-    parser.add_argument('--num-node-weights', required=True, type=int,
-                     help='The number of node weights used by METIS.')
-    parser.add_argument('--workspace', type=str, default='/tmp',
-                    help='The directory to store the intermediate results')
-    parser.add_argument('--node-attr-dtype', type=str, default=None,
-                    help='The data type of the node attributes')
-    parser.add_argument('--edge-attr-dtype', type=str, default=None,
-                    help='The data type of the edge attributes')
     parser.add_argument('--output', required=True, type=str,
                     help='The output directory of the partitioned results')
-    parser.add_argument('--removed-edges', help='a file that contains the removed self-loops and duplicated edges',
-                    default=None, type=str)
-    parser.add_argument('--exec-type', type=int, default=0,
-                    help='Use 0 for single machine run and 1 for distributed execution')
 
     #arguments needed for the distributed implementation
     parser.add_argument('--world-size', help='no. of processes to spawn',
                     default=1, type=int, required=True)
-    parser.add_argument('--nodes-file', help='filename of the nodes metadata', 
-                    default=None, type=str, required=True)
-    parser.add_argument('--edges-file', help='filename of the nodes metadata', 
-                    default=None, type=str, required=True)
-    parser.add_argument('--node-feats-file', help='filename of the nodes features', 
-                    default=None, type=str, required=True)
-    parser.add_argument('--edge-feats-file', help='filename of the nodes metadata', 
-                    default=None, type=str )
```

Comment:
why are `--edges-file`, `--node-feats-file` and `--edge-feats-file` no longer needed?

Reply:
These are no longer needed with the multiple-file-format support. In this new format, all the information is present in the metadata json file.